src/main/java/dk/cngroup/lentils/controller/HintController.java

```diff
@@ -136,18 +136,22 @@ public String saveNewHint(@Valid @ModelAttribute(TEMPLATE_ATTR_HINT) final HintF
             return VIEW_ADMIN_HINT_DETAIL;
         }
         Hint hint = new Hint(cypherService.getCypher(cypherId));
-        imageService.saveImageFile(hintFormDto);
         mapper.map(hintFormDto, hint);
         hintService.save(hint);
+        if (FileUtils.isFilePresentInHintForm(hintFormDto)) {
```

Comment:
Jen pro upřesnění (možná je součást jiné issue?). Momentálně teda backend nehlásí chybu, pokud uživatel nevybere soubor?

Reply:
Ne, neni duvod - file neni povinny, do db se priradi null a soubor se neuklada

Reply:
A jo, to me nenapadlo.